libs/ballot-interpreter-nh/src/images.test.ts

```diff
@@ -0,0 +1,202 @@
+import { createImageData } from 'canvas';
+import {
+  binarize,
+  getChannels,
+  matchTemplateImage,
+  simpleRemoveNoise,
+} from './images';
+import { Rect } from './types';
+import { loc } from './utils';
+
+/** foreground, represented as '#' in descriptions */
+const F = 0xff;
```

Comment:
This and `B` don't need to be so short. An earlier version of this used them directly rather than template strings, so shorter was better. Now there's no reason they can't be longer.